access-checkout-react-native-sdk/android/access-checkout-react-native-sdk-android-bridge/src/test/java/com/worldpay/access/checkout/reactnative/ui/AccessCheckoutTextInputManagerTest.kt

```diff
@@ -45,6 +46,19 @@ internal class AccessCheckoutTextInputManagerTest {
         verify(accessCheckoutEditTextMock).isEnabled = true
     }
 
+    @Test
+    fun `setRTCFont() should call textSize and typeface on AccessEditText`() {
+        val fontMap = JavaOnlyMap()
+        fontMap.putDouble("fontSize", 12.1)
+        fontMap.putString("fontFamily", "sans-serif")
+
+        val typeface = Typeface.create("sans-serif", Typeface.NORMAL)
+
+        manager.setRTCFont(accessCheckoutEditTextMock, fontMap)
+
+        verify(accessCheckoutEditTextMock).textSize = 12.1F
+        verify(accessCheckoutEditTextMock).typeface = typeface
+    }
```

Comment:
What would happen if we were to pass in a random string to fontFamily, a font that does not exist, should we add a test for this ?

Reply:
good shout, ill get that added

Reply:
From what I can see the create method will return the [best matching](https://developer.android.com/reference/kotlin/android/graphics/Typeface#create) typeface, in this case if the fontFamily we provide does not exist then it will return the default font.

Not sure of a way to test that since we cant access directly the fontFamily